ansible/roles/monasca/templates/monasca-log-api/log-api.conf.j2

```diff
@@ -33,3 +36,6 @@ service_token_roles_required=True
 memcache_security_strategy = ENCRYPT
 memcache_secret_key = {{ memcache_secret_key }}
 memcached_servers = {{ monasca_memcached_servers }}
+
+[elasticsearch]
+uris = {{ monasca_elasticsearch_servers }}
```

Comment:
Surprised that monasca doesn't need this to push logs too.

Reply:
They go direct to Kafka, and then logstash persists them to Elasticsearch.

Reply:
Ah, of course.